### DISCUSSION TITLE
side Effect

### DISCUSSION THREAD

**Discussion by mdgarden** [EasyAndBeauty/DailyReceipt-FE-WEB#11] [General]
#9 
뭔가 이런식의 리뷰는 한번도 못받아본것 같아요 🤔 시간되면 더 자세하게 들어보고 싶은..!

**Comment by IGhost-P** [comment]
사실 저 리뷰에서 좀 더 보안을 한다면 사용되는 변수들을 모두 매개변수로 받아와서 사용해야 한다 입니다..!
그렇게되면 매개변수로 들어오는 값이 달라지면 => 그에 맞게 결과값이 달라집니다
하지만 그렇지 않고 사용하게 되면 같은 매개변수가 들어왔지만 => 안에 사용되는 외부 변수가 달라져있다면? => 다른 결과값이 나옵니다.

이렇게 같은 입력에 같은 결과가 나오지 않는 경우가 부수효과 즉 side Effect라고 불립니다!
